output/compact_logs.go

```diff
@@ -16,6 +17,18 @@ func UploadAndSendLogs(server *state.Server, grant state.GrantLogs, collectionOp
 	if server.Config.FilterQuerySample == "all" {
 		logState.QuerySamples = []state.PostgresQuerySample{}
 	}
+	if server.Config.FilterQuerySample == "normalize" {
+		for idx, sample := range logState.QuerySamples {
+			// EXPLAIN was already handled separately, here we just need to normalize non-EXPLAIN samples
```

Comment:
Okay, I see--this case is handled after all. Given this package is called "output", though, I'm not sure if this is the best place for it...